src/views/PopupView/LoginPopup/LoginPopup.tsx

```diff
@@ -1,18 +1,18 @@
-import React, {useEffect, useState} from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
[minor]
Keep the code format. Same as above.

Reply:
I was fixed.